romana/cmd/tenant.go

```diff
@@ -108,28 +110,65 @@ func tenantCreate(cmd *cli.Command, args []string) error {
 		return err
 	}
 
-	tenants := []tenantData{}
+	tenants := []tenant.Tenant{}
 	for _, tnt := range args {
-		if !adaptor.TenantExists(tnt) {
-			// uncomment this once creating tenant for all
-			// platforms are ready. Till then tenants needs
-			// to be manually created for every platform and
-			// then created using romana command line tools.
-			// if adaptor.CreateTenant(tnt) != nil {
-			return err
-			// }
-		}
+		// uncomment this once creating tenant for all
+		// platforms are ready. Till then tenants needs
+		// to be manually created for every platform and
+		// then created using romana command line tools.
+		// if adaptor.CreateTenant(tnt) != nil {
+		// 	return err
+		// }
+		// adaptor.GetTenantUUID below wouldn't be needed once
+		// adaptor.CreateTenant is supported.
 		tntUUID, err := adaptor.GetTenantUUID(tnt)
 		if err != nil {
-			return err
+			switch err {
+			case util.ErrUnimplementedFeature:
+				// kubernetes doesnt support tenants yet so
```

Comment:
But it does support it: Namespaces.

How does this relate to our tenants then and where is that handled?